src/js/tech/html5.js

```diff
@@ -329,6 +333,7 @@ class Html5 extends Tech {
       }
     }
 
+    this.hasStartedPlayback_ = el.paused;
```

Comment:
`el.paused`, is also true if it's never played.

Reply:
yup, this should actually be `!el.paused`. Also @andriybohdan tested it and found that it just autoplayed everything, probably because of this.

Reply:
I think this might also be too late to check anyway. The element would be moved and apparently paused at https://github.com/videojs/video.js/blob/master/src/js/player.js#L643.

What is the disadvantage to creating a new video el as we do for iphone?

Reply:
Recreating the element in all conditions could potentially be a breaking change.
However, it'll be something to consider if this type of workaround wouldn't work.

Reply:
Also, I think this place is fine because we set the `autoplay` attribute a few lines above. Seems like `!el.paused` seems to work here but I guess we'd want to do more tests before pulling it in.